src/Model/Table/QueuedTasksTable.php

```diff
@@ -3,7 +3,7 @@
 
 use Cake\Core\Configure;
 use Cake\Utility\Hash;
-use Tools\Model\Table\Table;
+use Cake\ORM\Table;
```

Comment:
Well, you just need a second one on top of the existing one: https://github.com/dereuromark/cakephp-queue/blob/3.0/config/Migrations/20150425180802_init.php